resources/css/progress.css

```diff
@@ -25,22 +53,41 @@
   border-top: 0 solid;
 }
 
-.progressbar.player {
-  width: 104px;
+.completion-icon {
+  position: absolute;
+  top: -6.5px;
+  right: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 19px;
+  width: 19px;
+  border: solid 2px;
+  border-radius: 50%;
+  outline: solid 2px;
+  font-family: emoji;
+  font-size: 7pt;
```

Comment:
Use of `pt` here instead of `px/em/rem` is intentional?

Reply:
I've arrived at it after some trial-and-error. `9px` or `10px` produces similar results.

Reply:
Gotcha. Personally I'd prefer sticking to `px` for consistency but it doesn't matter too much in this case as the icons will most likely be replaced anyways.